serving/samples/autoscale-go/Dockerfile

```diff
@@ -14,14 +14,13 @@
 
 FROM golang AS builder
 
-WORKDIR /go/src/github.com/knative/docs/
-ADD . /go/src/github.com/knative/docs/
-
-RUN CGO_ENABLED=0 go build ./serving/samples/autoscale-go/
+WORKDIR /go/src/app
```

Comment:
If you're not going to put the source into a golang workspace, why not just put it in root?  E.g. `WORKDIR /app`.

Reply:
That's fair point. Just a convention to put into `/go/src` so folks can do `go install` within gopath. But I guess that has changed with go1.11. I think both are fien.